Header header
float64 resolution
Node[] nodes
Edge[] edges
